package.json

```diff
@@ -11,9 +11,8 @@
     "lint": "sk lint",
     "type-check": "sk type-check",
     "build": "sk build",
-    "version-bump": "lerna version --no-push --include-merged-tags",
     "version-bump:checkout": "lerna version --no-push --ignore-changes 'packages/{admin-ui-extensions*,ui-extensions}/**' --include-merged-tags",
-    "version-bump:admin": "lerna version --no-push --ignore-changes 'packages/!({admin*,ui-extensions})/**' --include-merged-tags",
+    "version-bump:admin": "lerna version --no-push --ignore-changes 'packages/!(admin*|ui-extensions)/**' --include-merged-tags",
```

Comment:
What does `|` do?

Reply:
admin* OR ui-extensions. I was just experimenting, and `!(a|b)` seems to work, but the old `!({a,b})` does not.

Reply:
the other ones are working with `{}`. I just noticed when I tried version bumping that the combo of `!` and `({})` weren't working for me.